lib/code_corps/policy/category.ex

```diff
@@ -1,9 +1,11 @@
 defmodule CodeCorps.Policy.Category do
-  alias CodeCorps.User
+  alias CodeCorps.{ User }
```

Comment:
We don't usually put whitespaces into brackets in elixir. I'd correct this myself, but you're using your own fork. Since it's just a single alias here, you don't really need brackets either.